internal/learn/future/shimmer.md

```diff
@@ -0,0 +1,34 @@
+---
+id: shimmer
+title: Shimmer
+description: Introduction to the emerging incentivized staging network
+keywords:
+  - Incentivized
+  - staking
+  - SMR
+---
+
+IOTA Foundation [introduced](https://blog.iota.org/introducing-iota-staking/) Shimmer. An incentivized staging network.
+It can be used to work with upcoming IOTA updates before they arrive on the mainnet. It will also have its own token called [SMR](https://shimmer.network/token).
+To fairly distribute tokens on the new network, you can stake your IOTA tokens before launch.
+
+## Pre-launch staking
+
+Before the Shimmer network goes online, the IOTA Foundation will announce a start date for "pre-launch" staking.
+You will be able to make a transaction to yourself with a certain payload to declare that you want to stake that amount of IOTA tokens to receive SMR. Staking will now be possible for 90 days. You will earn 1 SMR for every 1Mi every 10 seconds.
+There is no risk in staking your IOTA Tokens.
```

Comment:
```suggestion
During a 90 day period before the launch of the Shimmer network, you will be able to stake your IOTA tokens to receive SMR tokens on the Shimmer network upon launch. The IOTA Foundation will announce the start date 24 hours in advance.
During this 90 day period, you will earn 1 SMR for every 1Mi you staked every 10 seconds.
There is no risk in staking your IOTA tokens, you will remain in control of your tokens and won't have to transfer them to a third party.
```

What I tried to do here is describe the Pre-launch staking from a bit more abstract point of view and move the specifics down into the Technical overview.